domains/deepstate.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "nimrafer",
+    "email": "ntotsexk@hi2.in"
```

fake email address, if you don't want to share your email, you can safely remove this
```suggestion
    "username": "nimrafer"
```